src/js/actions/export.js

```diff
@@ -856,7 +878,7 @@ define(function (require, exports) {
                 .bind(this)
                 .then(function () {
                     log.debug("Export: Generator plugin connection established");
-                    return _setServiceAvailable.call(this, true);
+                    return this.flux.actions.export.setServiceAvailable(true);
```

Comment:
I suppose I get on board with (on both counts) not returning this "enqueue an action" promise.  However, the reason I'm doing this here is item No. 1 on the PR description.